packages/fields/src/types/CloudinaryImage/ImageBlock.js

```diff
@@ -1,7 +1,10 @@
 import pluralize from 'pluralize';
 import { Block } from '../../Block';
-import RelationshipFieldType from '../Relationship';
-import CloudinaryImage from './';
+
+const requireDefault = path => {
```

Comment:
Why is this necessary? It'll break when bundled with build-field-types

Reply:
Because we do a dynamic `require` call in the constructor to avoid circular dependencies. And since all the modules are now ESModules, it needs `.default` interop.

Why would it break? This is a server-side file doing a synchronous require.

Reply:
ESM should mean the circular dependency just work no? and when it's bundled, the circular dependency won't exist at all.

It would break because everything is compiled into a single bundle(except for things that are in their own chunks like views)

Reply:
Blow me down, you're right! ESModules have circular dependency resolution baked in. TIL 🎉

Now that I think about it; this was a problem before your refactor to use the new build system landed, and the `require()` calls were executed blindly by Node.

Just tested it with regular `import`s and it works 👍 

Have updated this branch.